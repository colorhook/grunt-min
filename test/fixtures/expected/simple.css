/*!
* comments
*/html{zoom:expression(function(ele){
   ele.style.zoom = "1";
   document.execCommand("BackgroundImageCache", false, true);
 }(this))}.break-all{-moz-hyphens:auto;-webkit-hyphens:auto;hyphens:auto}.fix-float{*zoom:1;_zoom:1;zoom:1\9}.fix-float:after{content:"\0020";display:block;height:0;clear:both}body{color:#3c3c3c;overflow-x:hidden;-webkit-backface-visibility:hidden;font:12px/1.5 "Lucida Grande",Helvetica,Arial,Verdana,'\5FAE\8F6F\96C5\9ED1',sans-serif}body[lang|=en]{color:#00f}.logo{background:url("data:image/png;base64,iVBj8+IDx4OnhsbnM6eD0kSuQmCC");*background:url("data:image/png;base64,iVBj8+IDx4OnhsbnM6eD0kSuQmCC")}@media (max-width:1280px){.nav{width:990px}}