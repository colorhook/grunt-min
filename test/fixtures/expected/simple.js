/*! a comment using *! protocol */
function min(n,e){return env+"."+name+"("+n+", "+e+");"}var env="grunt",name="grunt-min",result=min("a.js","a.min.js");