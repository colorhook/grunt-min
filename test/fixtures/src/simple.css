/*!
* comments
*/
html{
 zoom: expression(function(ele){
   ele.style.zoom = "1";
   document.execCommand("BackgroundImageCache", false, true);
 }(this));
}

.break-all{
 -moz-hyphens: auto;
 -webkit-hyphens: auto;
 hyphens: auto;
}

/* * identity */
.fix-float{
 *zoom:1;
 _zoom:1;
 zoom:1\9;
}

/* :status */
.fix-float:after{ 
 content:"\0020";
 display:block;
 height:0;
 clear:both
}

body{
 color: #3c3c3c;
 overflow-x: hidden;
 -webkit-backface-visibility: hidden;
 font: 12px/1.5 "Lucida Grande", Helvetica, Arial, Verdana, '\5FAE\8F6F\96C5\9ED1', sans-serif;
}

body[lang|=en] { 
  color:blue; 
}
/* datauri */
.logo{
  background:url("data:image/png;base64,iVBj8+IDx4OnhsbnM6eD0kSuQmCC");
}
/* media queries */
@media (max-width:1280px){
	.nav{
		width:990px;
	}
}