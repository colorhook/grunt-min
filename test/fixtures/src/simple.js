/*! a comment using *! protocol */
// a comment

var env = 'grunt';

var  name = 'grunt-min';

function min(src, output) {
  return env + '.' + name + '(' + src+ ', ' + output + ');';
}

var result = min('a.js', 'a.min.js');



