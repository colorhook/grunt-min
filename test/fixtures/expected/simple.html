<!DOCTYPE html><html lang="en-US"><head><meta charset="utf-8"><title>grunt-min</title><style>body{font-weight:bold;}</style></head><body><h1>grunt-min</h1><p>grunt-min is a grunt plugin for compressiong javascript, css, image & html</p></body></html>